tests/api_connexion/endpoints/test_logs/dag_for_testing_log_endpoint/task_for_testing_log_endpoint/2020-06-10T20.00.00+00.00/1.log

```diff
@@ -0,0 +1 @@
+Log for testing.
```

Comment:
I think this is left here by mistake.

Reply:
No. It is used by the tests

Reply:
This is a temporary file right? IMO it should be created in the start of the the test and should be unlinked during the tearDown. What do you think?

Reply:
I'm thinking it's going to be a lot of codes for the setUp method that's already bloated? What do you think? The file is just for testing. @mik-laj